spec/amber/validations/params_spec.cr

```diff
@@ -3,6 +3,21 @@ require "../../spec_helper"
 module Amber::Validators
   describe Params do
     describe "#validation" do
+      context "JSON param" do
+        it "parses json array as [] of JSON::Any" do
+          params = json_params_builder({ x: [ 1, 2, 3 ], y: {z: [ 1, 2 ] } }.to_json)
+          validator = Validators::Params.new(params)
+
+          validator.validation do
+            required(:x)
+            required("y")
+          end
+
+          validator.validate!["x"].should be_a Array(JSON::Any)
```

Comment:
Maybe returning a `JSON::PullParser` would be better here, so people could use  `JSON::Serializable` to decode these parameters. 🤔

Reply:
When was that introduced in the language? I don't recall seeing this at the time I wrote this PR.

Reply:
@a-alhusaini it's been part of the std library for a few years

https://crystal-lang.org/api/1.10.1/JSON/PullParser.html